lib/LTP/utils.pm

```diff
@@ -53,10 +55,13 @@ sub shutdown_ltp {
 }
 
 sub want_ltp_32bit {
+    my $pkg = shift // get_var('LTP_PKG');
+
     # TEST_SUITE_NAME is for running 32bit tests (e.g. ltp_syscalls_m32),
     # checking LTP_PKG is for install_ltp.pm which also uses prepare_ltp_env()
     return (get_required_var('TEST_SUITE_NAME') =~ m/[-_]m32$/
-          || get_var('LTP_PKG', '') =~ m/^(ltp|qa_test_ltp)-32bit$/);
+          || $pkg =~ m/^(ltp|qa_test_ltp)-32bit$/
```

Comment:
wouldn't it be simpler just `m/-32bit/` ?

Reply:
I didn't know why I wanted to be precise. I guess `m/-32bit$/` will do.